packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -351,6 +351,33 @@ describe("calcite-input-date-picker", () => {
       });
     });
 
+    it("closes when Escape key is pressed and focusTrapDisabled=true", async () => {
+      page = await newE2EPage();
```

Comment:
I think we could create a common test helper for focus trapping components. Can you create an issue for this?

Reply:
[Create a common test helper for focus trapping components](https://github.com/Esri/calcite-design-system/issues/10891#top)
#10891